cmd/down.go

```diff
@@ -15,7 +15,11 @@ var downCmd = &cobra.Command{
 }
 
 func downCommand(cmd *cobra.Command, _ []string) {
-	cfg, err := newConfigFromEnv()
+	composeFile, err := cmd.Flags().GetString("file")
```

Comment:
The bug we fixed in `up` hasn't been fixed here, we should probably add a function `func getFileFlag(cmd *cobra.Command) *string` to `cmd/common.go` that is called from here and from the up command.